Form-Controls/styles.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto";
+}
+.fxdc-jcc-aic {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.fx-jcc {
+  display: flex;
+  justify-content: center;
+}
+html {
+  height: 100vh;
+  background-color: rgb(209, 209, 209);
+}
+body {
+  background-color: black;
+  color: #fff;
+  border: 2px solid white;
+  border-radius: 1rem;
+  width: 18.75rem;
```

Comment:
My suggestion would be adding `width: 100%; height: 100%;` so the page extend to screen width and height

Reply:
Do you sure we have to put `width: 100%; height: 100%;` in the `body`?
![Screenshot from 2022-11-17 22-27-47](https://user-images.githubusercontent.com/107769663/202573666-8f1364d0-6ee3-45c0-9164-0ba18bdb36d7.png)

Reply:
You have to adjust your UI design accordingly. I am just suggesting that because if you do not fit the screen width and height, you cannot make a responsive page for all screen sizes.

Reply:
Now I get it. But this topic does not require RWD, because the question maker assigned this topic before the 3rd week, and the 2nd week did not start teaching RWD. Maybe you should review 4th week's code?

Reply:
Sure, I will review 4th week's code